### PR TITLE
Add empty HTTP server

### PR DESCRIPTION
**Pull request by thewilkybarkid**
Adds a HTTP server on port `8080` that responds with `200 OK` to every request.

### CODE REVIEW

File: .travis/smoke-test.sh [hunk 1]
```diff
@@ -8,3 +8,6 @@ function finish() {
 trap finish EXIT
 
 make start wait-healthy
+
+ping=$(curl -sS 'http://localhost:8080/' 2>&1)
+[[ $ping == 'OK' ]]
```
Comment (giorgiosironi) [inline .travis/smoke-test.sh:13]: is that the response body? Any value is good

Comment (thewilkybarkid) [reply .travis/smoke-test.sh:13]: Yeah, Koa's default when nothing is set.

Comment (giorgiosironi) [reply .travis/smoke-test.sh:13]: https://github.com/libero/article-store/pull/3/files#diff-f98485da1849ae631ba5c04e5cec07dd seems to say this is the response status?

Comment (thewilkybarkid) [reply .travis/smoke-test.sh:13]: Koa sets the body based on the status code.

Comment (thewilkybarkid) [reply .travis/smoke-test.sh:13]: Might be we should just test the code.

Comment (thewilkybarkid) [reply .travis/smoke-test.sh:13]: Possibly even redundant as it waits for the containers to be healthy? 🤔

Comment (giorgiosironi) [reply .travis/smoke-test.sh:13]: Reachable from the host is more than container being healthy as it tests also ports and how they are forwarded. Good as it is

**Review by giorgiosironi — APPROVED** [review]

**Pull request merged**